app/src/main/java/com/example/poste/models/Post.java

```diff
@@ -67,6 +76,24 @@ public String getUrl() {
      */
     public String getId() { return id; }
 
+    /**
+     * Deletes this post from the data model and backend
+     */
+    public void deletePost(Folder parentFolder) {
+        Post delPost = this;
+        MyApiService apiService = RetrofitClient.getRetrofitInstance().create(MyApiService.class);
+        Call<ResponseBody> call = apiService.deletePost(new DeletePost(delPost.getId()));
```

Comment:
Good start; remember we'll also need to pass the auth token in the header (see getData for more context)

Reply:
This too I will need a little more explanation on when we meet.